src/tools/tools_description.rs

```diff
@@ -421,13 +434,52 @@ impl ToolDict {
     }
 }
 
-pub fn tool_description_list_from_yaml(
+pub async fn get_custom_cmdline_tools(
+    gcx: Arc<ARwLock<GlobalContext>>,
+) -> Result<IndexMap<String, CustomCMDLineTool>, String> {
+    let tconfig = load_customization(gcx.clone(), true).await?;
+    let mut tools_dict = tconfig.custom_cmdline_tools.clone();
+    for (_, tool) in tools_dict.iter_mut() {
+        if tool.runs_in_background {
+            tool.description = format!("{}. Runs in background. Action is start by default", tool.description);
+            tool.parameters.push(AtParamDict {
+                name: "status".to_string(),
+                param_type: "boolean".to_string(),
+                description: "Action: get status of the background task. Path does not matter".to_string(),
+            });
+            tool.parameters.push( AtParamDict {
+                name: "restart".to_string(),
+                param_type: "boolean".to_string(),
+                description: "Action: restart the background task".to_string(),
+            });
+            tool.parameters.push( AtParamDict {
+                name: "stop".to_string(),
+                param_type: "boolean".to_string(),
+                description: "Action: restart the background task. Path does not matter".to_string(),
+            });
```

Comment:
not sure what that is, why 3 booleans instead of 1 string that is even easier to explain.